src/App.js

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <div className="App">
+    <h1>React sum</h1>
+
+    <Sum
+      a={2}
+      b={2}
+    />
```

Comment:
No need to add new lines here, code is short enough.
```suggestion
    <Sum a={2} b={2} />
```